src/main/resources/features/moth/54 - Print MOT certificate with failures.feature

```diff
@@ -0,0 +1,37 @@
+@mothprint @mothint
+Feature: 54 - Check whether MOT certificates can be downloaded after entering the V5C number
+
+  Scenario: A MOTH user searches for a vehicle with an MOT test and downloads the MOT certificate
```

Comment:
The description for the feature and scenario doesn't match what the tests are doing, can this be changed.

Reply:
The feature description and scenario changed to match what is happening in the tests.